packages/sanity/src/tasks/src/tasks/components/create/TasksCreate.tsx

```diff
@@ -1,11 +1,12 @@
 import {useCallback, useEffect, useRef, useState} from 'react'
-import {Box, Card, Flex, Stack, Text, TextInput, useToast} from '@sanity/ui'
+import {Card, Flex, Stack, Text, TextInput, useToast} from '@sanity/ui'
 import {PortableTextBlock} from '@sanity/types'
 import {Button} from '../../../../../ui-components'
 import {useTasks} from '../../context'
 import {CommentInput, useMentionOptions} from '../../../../../structure/comments'
```

Comment:
Can we import from structure into core? Not sure that is possible, meaning we might need refactor these once they go into the main tasks branch.